package.json

```diff
@@ -21,7 +21,7 @@
         "jshint": ">=1.1.0",
         "mocha": ">=1.9.0",
         "mocha-istanbul": ">=0.2.0",
```

Comment:
Agreed. I was just thinking about that last night when I was working on this PR.